src/fal/el/airbyte.py

```diff
@@ -0,0 +1,129 @@
+"""Synchronize with Airbyte."""
+from dataclasses import dataclass, field
+from typing import Any, Dict, Optional
+import requests
+import time
+from requests.exceptions import RequestException
+from dbt.logger import GLOBAL_LOGGER as logger
+
+
+class AirbyteJobState:
+    """Possibe Airbyte job states."""
+
+    RUNNING = "running"
+    SUCCEEDED = "succeeded"
+    CANCELLED = "cancelled"
+    PENDING = "pending"
+    FAILED = "failed"
+    ERROR = "error"
+    INCOMPLETE = "incomplete"
+
+
+@dataclass
+class AirbyteClient:
+    """Airbyte REST API connector class."""
+
+    host: str
```

Comment:
is there a cloud solution from airbyte we could point to by default?

Reply:
Airbyte cloud doesn't have API access yet.